packages/@react-types/button/src/index.d.ts

```diff
@@ -20,6 +20,20 @@ interface ButtonProps extends PressEvents, FocusableProps {
   children?: ReactNode
 }
 
+type RelOption =
```

Comment:
here's a full list: https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel

i don't think react goes further than `rel?: string;` in its typedefs, so it's uncertain to me if we'd want to be even more specific in this library.

Reply:
I've taken the full list that is valid for an a element from that page.

But if we want to keep it to just a string, I'm fine with that too 😊.

Reply:
i specifically noticed the missing `nofollow` since that's the one i use 😅 sorry for the nit

Reply:
I think we should try to be consistent with the DOM here just so it's not confusing for people.